sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -0,0 +1,352 @@
+	local x
+	for x in ${VTK_DROP[@]}; do
+		elog "Dropping bundled ${x}"
+		rm -rf ThirdParty/${x}/vtk${x} || die "failed to drop ${x}"
+	done
+
+	if use doc; then
+		elog "Removing .md5 files from documents."
+		rm -f "${WORKDIR}"/html/*.md5 || die "Failed to remove superfluous hashes"
+		sed -e "s|\${VTK_BINARY_DIR}/Utilities/Doxygen/doc|${WORKDIR}|" \
+			-i Utilities/Doxygen/CMakeLists.txt || die
+	fi
+}
+
+src_configure() {
+	# general configuration
+	local mycmakeargs=(
+		-Wno-dev
+		-DVTK_DIR="${S}"
+		-DVTK_DATA_ROOT="${EPREFIX}/usr/share/${PN}/data"
+		-DBUILD_DOCUMENTATION=$(usex doc)
+		-DBUILD_EXAMPLES=$(usex examples)
+		-DVTK_MODULE_USE_EXTERNAL_utf8cpp=ON
+		-DVTK_USE_SYSTEM_AUTOBAHN=ON
+		-DVTK_USE_SYSTEM_EXPAT=ON
+		-DVTK_USE_SYSTEM_FREETYPE=ON
+		-DVTK_USE_SYSTEM_FreeType=ON
+		-DVTK_USE_SYSTEM_GL2PS=ON
+		-DVTK_USE_SYSTEM_HDF5=ON
+		-DVTK_USE_SYSTEM_JPEG=ON
+		-DVTK_USE_SYSTEM_LIBPROJ=OFF
+		-DVTK_USE_SYSTEM_LIBXML2=ON
+		-DVTK_USE_SYSTEM_LibXml2=ON
+		-DVTK_USE_SYSTEM_NETCDF=ON
+		-DVTK_USE_SYSTEM_OGGTHEORA=ON
+		-DVTK_USE_SYSTEM_PNG=ON
+		-DVTK_USE_SYSTEM_TIFF=ON
+		-DVTK_USE_SYSTEM_TWISTED=ON
+		-DVTK_USE_SYSTEM_XDMF2=ON
+		-DVTK_USE_SYSTEM_XDMF3=OFF
+		-DVTK_USE_SYSTEM_ZLIB=ON
+		-DVTK_USE_SYSTEM_ZOPE=ON
+		-DVTK_USE_SYSTEM_LIBRARIES=ON
+		-DVTK_USE_SYSTEM_DIY2=ON
+		-DVTK_USE_GL2PS=ON
+		-DVTK_USE_LARGE_DATA=ON
+		-DVTK_USE_PARALLEL=ON
+		-DVTK_EXTRA_COMPILER_WARNINGS=ON
+		-DVTK_Group_StandAlone=ON
+		-DBUILD_VTK_BUILD_ALL_MODULES_FOR_TESTS=off
+		-DVTK_BUILD_ALL_MODULES=$(usex all-modules)
+		-DUSE_DOCUMENTATION_HTML_HELP=$(usex doc)
+		-DVTK_Group_Imaging=$(usex imaging)
+		-DVTK_Group_MPI=$(usex mpi)
+		-DVTK_Group_Rendering=$(usex rendering)
+		-DVTK_Group_Tk=$(usex tk)
+		-DVTK_Group_Views=$(usex views)
+		-DVTK_Group_Web=$(usex web)
+		-DVTK_SMP_IMPLEMENTATION_TYPE="$(usex tbb TBB Sequential)"
+		-DVTK_WWW_DIR="${ED}/${MY_HTDOCSDIR}"
+		-DVTK_WRAP_JAVA=$(usex java)
+		-DVTK_WRAP_PYTHON=$(usex python)
+		-DVTK_WRAP_PYTHON_SIP=$(usex python)
+		-DVTK_WRAP_TCL=$(usex tcl)
+		-DVTK_USE_BOOST=$(usex boost)
+		-DUSE_VTK_USE_BOOST=$(usex boost)
+		-DModule_vtkInfovisBoost=$(usex boost)
+		-DModule_vtkInfovisBoostGraphAlgorithms=$(usex boost)
+		-DVTK_USE_ODBC=$(usex odbc)
+		-DModule_vtkIOODBC=$(usex odbc)
+		-DVTK_USE_OFFSCREEN=$(usex offscreen)
+		-DVTK_OPENGL_HAS_OSMESA=$(usex offscreen)
+		-DVTK_USE_OGGTHEORA_ENCODER=$(usex theora)
+		-DVTK_USE_NVCONTROL=$(usex video_cards_nvidia)
+		-DModule_vtkFiltersStatisticsGnuR=$(usex R)
+		-DVTK_USE_X=$(usex X)
+	# IO
+		-DVTK_USE_FFMPEG_ENCODER=$(usex ffmpeg)
+		-DModule_vtkIOGDAL=$(usex gdal)
+		-DModule_vtkIOGeoJSON=$(usex json)
+		-DModule_vtkIOXdmf2=$(usex xdmf2)
+		-DBUILD_TESTING=$(usex examples)
+	# Apple stuff, does it really work?
+		-DVTK_USE_COCOA=$(usex aqua)
+	)
+
+	if use java; then
+		local javacargs=$(java-pkg_javac-args)
+		mycmakeargs+=( -DJAVAC_OPTIONS=${javacargs// /;} )
+	fi
+
+	if use mpi; then
+		mycmakeargs+=( -DVTK_USE_SYSTEM_MPI4PY=ON )
```

Comment:
I think this should be nested into the if use python below
See also line 92

Reply:
There are so many changes to make, reviewing and commenting right now is not that useful.
Basically, most options and workings are going to be replaced.

Reply:
Yeah, I take non-draft to mean "ready for comments". Let her work for now. I use GitHub PRs like this for CI and to keep the work safe.